Changed all readme docs -- just missing a couple of collaborators because I do not know their github profiles (#11)
dcl-covid-19/mega-map-dev: docs/content-editors.md

# Bay Area Mega Map Content Editors <!-- omit in toc -->

Created by the Bay Area Mega Map team 

Pieces of this code has been adapted from [Code for Chapel Hill](http://www.codeforchapelhill.com/) part of [Code for America](https://www.codeforamerica.org/)

## Introduction <!-- omit in toc -->

> Thank you for your interest in helping to add content to the Bay Area Mega Map website, we really appreciate everyone who is able to volunteer their time, wether that be to update a single entry or to manage a whole section of our database.

Reviewer: ```suggestion
Thank you for your interest in helping to add content to the Bay Area Community Resources website, we really appreciate everyone who is able to volunteer their time, whether that be to update a single entry or to manage a whole section of our database.
```

## Team Structure

This project is the accummulation of the tireless work done by a team of more than 60 volunteers coordinated by Charlie Hoffs

The dev team on this project is led by [Maeva Fincker](https://github.com/mfincker)

## Project Lead

> [Joyce Tagal](https://github.com/jayktee) is the project lead for this work and questions or feedback should be directed to her

Reviewer: ```suggestion
[Joyce Tagal](https://github.com/jayktee) is the project lead for this project. Any questions or feedback should be directed to her.
```

### Content Ambassadors

> Are you able to spare some time to become a volunteer  and help improve / keep the resources or map up to date? We are looking for people to join our enthusiastic team of ambassadors so that we are able to keep this resource up to date through the covid-19 pandemic.  

Reviewer: ```suggestion
Are you able to spare some time to become a volunteer  and help improve / keep the resources or map up to date? We are looking for people to join our enthusiastic team of ambassadors so that we are able to keep this resource up to date through the Covid-19 pandemic.  
```
